packages/react/src/components/DataTable/TableToolbar.js

```diff
@@ -4,16 +4,33 @@
  * This source code is licensed under the Apache-2.0 license found in the
  * LICENSE file in the root directory of this source tree.
  */
+
+import PropTypes from 'prop-types';
 import React from 'react';
 import { settings } from 'carbon-components';
 
 const { prefix } = settings;
 
-const TableToolbar = ({ children }) => (
-  <section
-    aria-label="data table toolbar"
-    className={`${prefix}--table-toolbar`}>
+const TableToolbar = ({ ariaLabel, children }) => (
+  <section aria-label={ariaLabel} className={`${prefix}--table-toolbar`}>
```

Comment:
Probably propagating getting the rest in the args list and spread it here, instead of just supporting `aria-label`?

Reply:
Thanks, I'll do that!